src/local-settings.js

```diff
@@ -96,3 +98,12 @@ export function getIpfsRpc() {
     }
   }
 }
+
+export function getSelectedCurrency() {
+  const currency = getLocalSetting(SELECTED_CURRENCY, 'USD')
+  return currency && currency.toUpperCase()
```

Comment:
I think this can just be `return currency.toUpperCase()`; the default `'USD'` will be returned in any case where the locally saved version is false.